lab08-jacob/test/route-test.js

```diff
@@ -0,0 +1,67 @@
+//one it block per route. Do a fetch and then say it should yadda yadda
+const request = require('superagent');
+const expect = require('chai').expect;
+
+require('../server.js');
+var user = null; //I only user this for the non-persistence test
```

Comment:
Does it need to be a global then, or can it be defined within the test that uses it?